src/python/bot/tasks/fuzz_task.py

```diff
@@ -202,6 +202,12 @@ def __init__(self,
     self.crash_frames = state.frames
     self.crash_info = None
 
+    # Fuzzing log containing strategy is stored here.
+    self.fuzzing_log = ''
```

Comment:
Sorry for the delayed review! I don't think this is the right way to fix this. See my other comments.